input/fsh/profiles/EncounterBirth.fsh

```diff
@@ -42,7 +42,7 @@ Description: "This Encounter profile represents the child's birth encounter."
 * location MS
   * ^short = "Information about the location of birth"
   * ^definition = "Information about the location of birth"
-  * location only Reference(USCoreLocation)
+  * location only Reference(LocationBFDR)
   * location MS
     * ^short = "Facility name (if not institution, give street and number)"
     * ^definition = "Facility name (if not institution, give street and number)"
```

Comment:
"HOSP" is mapped to location.location.name - we should have a short description in encounter birth so it shows in the differential